docs/source/Getting-Started/Keystone-Security-Monitor.rst

```diff
@@ -1,135 +1,146 @@
 Keystone Security Monitor
 ====================================
 
-The security monitor (SM) is a trusted software that runs in M-mode, acting as a trusted computing
+The security monitor (SM) is trusted software that runs in M-mode, and forms the trusted computing
 base (TCB) in Keystone system.
-SM is the most important software in Keystone system for the security guarantees.
-SM provides the following functionalities:
+The SM is the most important software in Keystone as it provides most of the security guarantees.
+The SM provides the following functionality:
 
 1. Memory Isolation using RISC-V PMP
-2. Remote Attestation
+2. Remote Attestation (Signatures and measurement)
 3. Enclave Thread Management
 4. System PMP Synchronization
-5. Side-Channel Defense
+5. Side-Channel Defenses
 
 .. attention::
 
   Version 0.1 only implements 1, 2, and 3, supporting single-threaded enclave.
 
 The security monitor must be verified by the *root of trust* in hardware before being trusted.
-After the root of trust verifies SM, it starts booting with a keypair for remote attestation, 
-where the public key was signed by the root of trust.
-See `Chain of Trust <#>`_ for how it works.
+After the root of trust measures the SM it generates a keypair for remote attestation, signs the public key, and then continues booting.
+See `Chain of Trust <#>`_ for details.
 
-The OS or the enclaves can call SM functions using supervisor binary interface (SBI), 
-which is implemented by environment call (i.e., ``ecall`` instruction).
+The OS and enclaves may call SM functions using the supervisor binary interface (SBI),
+which is implemented in RISC-V environment calls (i.e., ``ecall`` instruction).
 See `SBI Documentation <#>`_ for details.
 
+PMP Internals
+-------------
+
+PMP allows for N (platform defined) entries, which are configured
+using a config (cfg) and address (addr) register each. There are
+further details available in the RISC-V `specification
+<https://content.riscv.org/wp-content/uploads/2017/05/riscv-privileged-v1.10.pdf>`
```

Comment:
you need to add _ at the end to make it a link
e.g., 
`specification <https://content.riscv.org/wp-content/uploads/2017/05/riscv-privileged-v1.10.pdf>`_